DataFormats/Detectors/EMCAL/include/DataFormatsEMCAL/CellLabel.h

```diff
@@ -34,10 +35,15 @@ class CellLabel
  public:
   // CellLabel() = default;
 
-  /// \brief Constructor
+  /// \brief Constructor using std::vector
   /// \param labels list of mc labels
   /// \param amplitudeFractions list of amplitude fractions
-  CellLabel(const gsl::span<const int> labels, const gsl::span<const float> amplitudeFractions);
+  CellLabel(std::vector<int> labels, std::vector<float> amplitudeFractions);
```

Comment:
what is the reason for removing the const?

Reply:
Changed to a move constructor. No more copies will be made.